django_forest/utils/forest_api_requester.py

```diff
@@ -0,0 +1,84 @@
+import json
+import os
+from urllib.parse import urljoin
+
+import requests
+from django.conf import settings
+
+
+class ForestApiRequester:
+
+    @staticmethod
+    def get_headers(headers):
+        forest_secret_key = getattr(settings, 'FOREST', {}).get('FOREST_ENV_SECRET', os.getenv('FOREST_ENV_SECRET'))
+        base_headers = {
+            'Content-Type': 'application/json',
+            'forest-secret-key': forest_secret_key,
+        }
+        base_headers.update(headers)
+        return base_headers
+
+    @staticmethod
+    def error_msg(url):
+        return f'Cannot reach Forest API at {url}, it seems to be down right now.'
+
+    @staticmethod
+    def forest_api_url():
+        return getattr(settings, 'FOREST', {}).get('FOREST_URL', os.getenv('FOREST_URL', 'https://api.forestadmin.com'))
+
+    @classmethod
+    def _get_url(cls, route):
+        if route.startswith('https://'):
```

Comment:
What if route starts by `http://`?

Reply:
I don't know... This is something weird I have seen in `forest-rails` : https://github.com/ForestAdmin/forest-rails/blob/master/app/services/forest_liana/forest_api_requester.rb#L19
I don't find a similar thing in `forest-express`...